build-pipeline/aspnet-docker-windows-images.json

```diff
@@ -204,7 +204,8 @@
     }
   },
   "demands": [
-    "Agent.OS -equals Windows_NT"
+    "VSTS_OS -equals Windows_Server_2016_Data_Center_with_Containers",
+    "Agent.Name -equals DDVSOWINAGE106"
```

Comment:
@ravimeda will more agents be added to "Windows_Server_2016_Data_Center_with_Containers", or should I keep this agent filter?

Reply:
I can't comment if more agents will be added but the Agent.Name demand should be removed so that as agents come/go the build will continue to work.

Reply:
A few more agents will help reduce the build leg wait time.  Please log a ticket with DDIT as needed.